src/scf/driver/scf_loop.cpp

```diff
@@ -81,47 +89,59 @@ MODULE_CTOR(SCFLoop) {
     add_input<double>("density tolerance").set_default(1.0E-6);
     add_input<double>("gradient tolerance").set_default(1.0E-6);
 
+    const std::size_t diis_sample_default = 5;
+    add_input<bool>("DIIS").set_default(false);
```

Comment:
I'd suggest we default to DIIS on (we're not at a 1.0 on SCF and can change the default behavior).
```suggestion
    add_input<bool>("DIIS").set_default(true);
```